model/browser/events.yaml

```diff
@@ -52,3 +52,70 @@ groups:
           brief: A unique ID representing this particular metric instance.
           requirement_level: required
           examples: ["v3-1677874579383-6381583661209"]
+
+  - id: event.browser.page_view
+    stability: development
+    type: event
+    name: browser.page_view
+    brief: >
+      This event represents a browser page load/view.
+    note: >
+      The primary use case is to capture metrics about the number of views of
+      a page. Page view can be a hard page load in a browser as well as virtual
+      navigation in a SPA (single-page application). The event should be
+      collected as soon as possible without waiting for potentially delayed
+      data such as the Navigation timing browser API.
+    body:
```

Comment:
Should all of this be moved out of the body and instead be set as attributes? Or one complex attribute?